src/App.tsx

```diff
@@ -1,77 +1,9 @@
-import React, { useState } from 'react';
-import './App.scss';
-import GenreSelect from './components/GenreSelect/GenreSelect';
-import Header from './components/Header/Header';
-import MovieTile from './components/MovieTile/MovieTile';
-import SortControl from './components/SortControl/SortControl';
-import { genresMock } from './mocks/Genre';
-import { movieMock, movieMock2, movieMock3 } from './mocks/Movie';
-import { Movie } from './models/Movie';
+import React from 'react';
+import MovieListPage from './pages/MovieListPage/MovieListPage';
 
 function App() {
-  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
-  const movies = [movieMock, movieMock2, movieMock3];
-  const genres = ['All', ...genresMock];
-
-  const handleSearch = (text: string) => {
-    console.log('searching for item...', text);
-  }
-
-  const handleGenreSelect = (genre: string) => {
-    console.log('genre selected', genre);
-  }
-
-  const handleSortChange = (sort: string) => {
-    console.log('sort changed', sort);
-  }
-
-  const handleTileClick = (movie: Movie) => {
-    setSelectedMovie(movie);
-  }
-
-  const handleAddMovieSubmit = (movie: Movie) => {
-    console.log('Add movie submitted', movie);
-  }
-
-  const handleEditMovieSubmitted = (movie: Movie) => {
-    console.log('Edit movie submitted', movie);
-  }
-
-  const handleDeleteClick = (id: string) => {
-    console.log('Delete movie clicked', id);
-  }
-
-  const handleSelectedMovieChange = (movie: Movie | null) => {
-    setSelectedMovie(movie);
-  }
-
   return (
```

Comment:
I removed the closing tag, but if you don't mind I would like to leave `handleSelectedMovieChange` intact because I will need it in the next lesson :) 
And as always: great review, thank you!